src/app/cpe/page/page.component.ts

```diff
@@ -38,9 +37,14 @@ export class PageComponent implements OnInit, OnDestroy {
    * @param pageId: The id of the page to be loaded.
    */
   loadPage(pageId: string) {
-    this._pageService.getPageById(pageId).then(p => {
-        this.page = p;
-    }).catch(err => console.error(err) );
+    this._pageService.getPageById(pageId).subscribe({
+        next: (page) => {
```

Comment:
here and everywhere else, you can omit `next:` and `error:` if you'd like to make the code a bit simpler

Reply:
I did it for @domsteinbach, so he sees where errors can be handled and can continue from there